app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+  <p>Tax-free scholarships of £27,000 and bursaries of £25,000 are available for eligible trainee language teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "",
+      colour: "white",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+  
+    <% end %>
+  </section>
+
+    <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "As a language teacher, you can light the flame of language learning and build cultural capital for all young people",
+    name: "Teacher",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You will follow the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-languages-progammes-of-study">national curriculum. For any modern language you teach, you will:</p>
+<ul>
+  <li>prepare lessons and teaching materials </li>
+<li>assess pupils' progress and mark their work</li>
+<li>set homework </li>
+<li>manage classroom behaviour </li>
+<li>organise study trips, social activities and sports events</li>
+<li>Monitor the progress of pupils, through coursework and homework </li>
+<li>Work with pupils to guide their development and share that progress with parents and caregivers </li>
+<li>Plan, supervise and lead study trips</li>
+</ul>
+      <p><b>Themes you’ll cover for 11 to 14 year olds (key stage 3):</b></p>
+
+<p>You'll help pupils develop their understanding of grammar and vocabulary and develop their linguistic competence, by helping them to:  </p>
+
+<ul>
+<li>identify and use present, past, and future tenses as appropriate</li>
+
+<li>use a variety of key grammatical structures and patterns like voices and moods</li>
+
+<li>develop and use a wide-ranging and deepening vocabulary, to take part in discussions about wider issues</li>
+
+<li>use accurate grammar, spelling and punctuation</li>
+    </ul>
+
+<p>To develop pupil's linguistic competence skills at key stage 3, you will teach them to: </p>
+
+<ul>
+<li>listen to a variety of forms of spoken language to obtain information and respond appropriately</li>
+
+<li>transcribe words and short sentences that they hear with increasing accuracy</li>
+
+<li>have conversations and make use of important social conventions such as formal modes of address</li>
+
+<li>express and develop ideas clearly and with increasing accuracy, both orally and in writing</li>
+
+<li>speak coherently and confidently</li>
+
+<li>read literary texts in the language and expand their understanding of the language and culture</li>
+
+<li>write creatively to express their own ideas and opinions</li>
+</ul>
+
+You can explore:
```

Comment:
@caitlin-reidy You can explore examples of: - don't need examples after each bullet point